daemon/algod/api/server/v2/errors.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2019-2020 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package v2
+
+var (
+	errFailedLookingUpLedger            = "failed to retrieve information from the ledger"
+	errFailedLookingUpTransactionPool   = "failed to retrieve information from the transaction pool"
+	errFailedRetrievingNodeStatus       = "failed retrieving node status"
+	errFailedParsingFormatOption        = "failed to parse the format option"
+	errFailedToParseAddress             = "failed to parse the address"
+	errFailedToParseTransaction         = "failed to parse transaction"
+	errFailedToParseBlock               = "failed to parse block"
+	errFailedToParseCert                = "failed to parse cert"
+	errInternalFailure                  = "internal failure"
+	errNoTxnSpecified                   = "no transaction ID was specified"
+	errTransactionNotFound              = "couldn't find the required transaction in the required range"
```

Comment:
The only place this error is used is when the required txid is not found in pending transactions endpoint. Since the user doesn't have the option to specify any range, this is probably not the right message to provide him with.